.github/workflows/release-chart.yml

```diff
@@ -0,0 +1,48 @@
+name: Release newrelic infrastructure chart
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+#   Sometimes chart-releaser might fetch an outdated index.yaml from gh-pages, causing a WAW hazard on the repo
+#   This job checks the remote file is up to date with the local one on release
+  validate-gh-pages-index:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: gh-pages
+      - name: Download remote index file and check equality
+        run: |
+          curl -vsSL https://newrelic.github.io/nri-kubernetes/index.yaml > index.yaml.remote
```

Comment:
I would not change it since every-other pipeline has it hardcoded. Moreover, it is impossible not to notice it in case you mistype it since the pipeline would fail